tools/checkpoint_conversion/convert_gemma_checkpoints.py

```diff
@@ -86,7 +86,24 @@ def download_flax_model(handle):
     return kagglehub.model_download(handle)
 
 
-def convert_model(flax_config, vocab_size):
+def convert_model(flax_config, flax_params, vocab_size):
+    kwargs = {}
+    # Hack to infer Gemma 2 config options until Flax actually adds support.
+    if "post_attention_norm" in flax_params["transformer"]["layer_0"]:
+        # The 27B parameter model is the only model that does a weird
+        # query normalization.
+        query_head_dim_normalize = flax_config.num_heads != 32
+        # We would like to convert these from Flax, but have no way until
+        # flax supports Gemma 2.
+        kwargs = {
+            "query_head_dim_normalize": False,
```

Comment:
Should this be `query_head_dim_normalize` variable defined above instead of `False`? I think that's why the test is complaining!

Reply:
yep!